HISTORY.md

```diff
@@ -1,3 +1,57 @@
+# TileDB v2.21.0 Release Notes
+
+## Deprecation announcements
+
+* Windows 2019 build support will be removed in version 2.28, to be released in Q3 2024.
```

Comment:
Is this Windows 2019 or Visual Studio 2019? To be more precise we should specify the minimum target toolchain version.

Reply:
Updated my suggestion above.

Reply:
Thanks